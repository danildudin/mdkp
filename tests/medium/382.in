29 6
491 671 159 600 230 68 58 484 266 432 116 90 788 834 774 660 308 404 19 988 155 904 131 372 158 984 140 793 859 
177 566 501 992 627 629 92 260 578 969 380 204 471 51 68 177 549 532 839 913 566 947 704 40 308 112 85 904 190 9581
369 770 50 744 149 31 463 551 57 511 989 974 133 984 699 517 551 101 223 453 288 443 36 868 593 784 561 128 148 8178
265 875 260 779 200 675 561 889 358 922 350 268 902 262 304 523 28 856 712 337 980 757 839 370 389 19 952 343 910 9210
576 477 595 876 552 638 930 209 97 424 81 269 342 767 328 635 254 997 83 922 479 379 8 515 806 782 530 855 907 8680
483 201 994 54 629 812 811 493 11 874 475 771 912 317 109 314 165 455 814 358 488 365 751 680 747 377 843 889 752 7306
74 484 946 586 878 174 20 290 255 488 137 981 370 74 257 237 340 588 836 788 297 453 579 432 840 866 38 84 798 9734
