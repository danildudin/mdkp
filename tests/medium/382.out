10608
16
0 1 7 8 9 12 13 14 15 16 19 21 23 25 27 28 
